projects/vitejs.dev/package.yml

```diff
@@ -1,22 +1,27 @@
-distributable: ~
+distributable: https://github.com/vitejs/vite/archive/refs/tags/v{{version.raw}}.tar.gz
 
 versions:
   github: vitejs/vite/tags
   strip: /v/
 
+dependencies:
+  pnpm.io: "*"
```

Comment:
this is not a runtime dependency surely?

Reply:
Moved to `build`